packages/next-app/src/components/Table.tsx

```diff
@@ -77,7 +77,7 @@ function HeaderCell({ children }: React.PropsWithChildren) {
   return (
     <th
       scope="col"
-      className="text-gray-40 p-4 text-left text-sm font-semibold"
+      className="text-gray-40 px-1 py-4 text-left text-sm font-semibold"
```

Comment:
You're changing the component everywhere -- just a reminder to ensure it looks nice across all uses.

Reply:
I will test it!